host_v2/src/components/ConfidenceGraph/ConfidenceResponseDropdown.tsx

```diff
@@ -133,43 +118,24 @@ export default function ConfidenceResponseDropdown({
    * @param selectedData confidence data passed in from parent
    * @returns sorted array of input
    */
-  const sortPlayers = (selectedData: {
-    players: Player[];
-  }): {
-    correct: Player[];
-    incorrect: Player[];
+  const sortPlayers = (selectedConfidenceInput: IHostTeamAnswersConfidence): {
+    correct: IHostTeamAnswersConfidenceResponse[];
+    incorrect: IHostTeamAnswersConfidenceResponse[];
   } => {
-    const correctPlayers: Player[] = [];
-    const incorrectPlayers: Player[] = [];
+    const correctPlayers: IHostTeamAnswersConfidenceResponse[] = selectedConfidenceInput.correct;
+    const incorrectPlayers: IHostTeamAnswersConfidenceResponse[] = selectedConfidenceInput.incorrect;
     const answerFrequency: Record<string, number> = {};
-    selectedData.players.forEach((playerData: Player) => {
-      // split players into correct and incorrect so .sort is limited to these subsets
-      if (playerData.isCorrect) {
-        correctPlayers.push(playerData);
-      } else {
-        incorrectPlayers.push(playerData);
-        // if incorrect, also store the frequency of the answer for sorting later
-        answerFrequency[playerData.answer] =
-          (answerFrequency[playerData.answer] || 0) + 1;
-      }
-    });
+  
     // sort correct alphabetically
-    correctPlayers.sort((a, b) => a.name.localeCompare(b.name));
-    incorrectPlayers.sort((a, b) => {
-      // sort incorrect by answer frequency, then alphabetically
-      const freqDifference =
-        answerFrequency[b.answer] - answerFrequency[a.answer];
-      return freqDifference !== 0
-        ? freqDifference
-        : a.name.localeCompare(b.name);
-    });
+    correctPlayers.sort((a, b) => a.team.localeCompare(b.team));
+    incorrectPlayers.sort((a, b) => a.team.localeCompare(b.team));
```

Comment:
nit: Since these two are the same sort logic, we can create a local variable for the sort method and then pass it here as input